### PR TITLE
Ecs_node role refactoring

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Ecs_node role can create one single node with defined name and ip address and a few nodes with indexed names (before there was only second choice)

### CODE REVIEW

File: roles/ecs_node/tasks/build.yaml [hunk 1]
```diff
@@ -74,3 +76,5 @@
     auto_ip: false
   with_sequence: start=1 end="{{ ecs_count|int }}"
   register: instances
+
+
```
Comment (anton-sidelnikov) [inline roles/ecs_node/tasks/build.yaml]: excess string

File: roles/ecs_node/vars/main.yaml [hunk 2]
```diff
@@ -1 +1,3 @@
 az_count: "{{ ecs_availability_zones | length }}"
+# ecs_name:  if set - one single node will be created, if not set - nodes amount = ecs_count
+# ecs_ip_address: if set ecs_name - this address will be assigned to node
```
Comment (anton-sidelnikov) [inline roles/ecs_node/vars/main.yaml]: no empty line

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: roles/ecs_node/vars/main.yaml (hunk 2 above)
Comment (Polina-Gubina) [reply roles/ecs_node/vars/main.yaml]: done

File: roles/ecs_node/tasks/build.yaml (hunk 1 above)
Comment (Polina-Gubina) [reply roles/ecs_node/tasks/build.yaml]: done

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**